### PR TITLE
feat: identities page rework

### LINKED ISSUE #423: Long loading not showing as a loading (skeleton)

### LINKED ISSUE #428: Identities page
Loom explication : https://www.loom.com/share/e9fffd22769a466e976d846e8e468c02?sid=99b62576-cddd-45fc-b139-0abff5415c01

Figma : https://www.figma.com/file/S1UKYgWewNqNHZFAaBUilG/Starknet-ID?type=design&node-id=1819%3A15095&mode=design&t=3HLgykmyOy6S15zF-1

### PR DESCRIPTION
**Pull request by Marchand-Nicolas**
close: #428
close: #423

### CODE REVIEW

**Review by fricoben — CHANGES_REQUESTED** [review]
Capture d’écran 2023-10-08 à 10 59 12

Size is different if domain or not, maybe you cann add the tokenId like `ID: XXX`

Capture d’écran 2023-10-08 à 10 55 06

width too big

braavos external domains not showing as on the design (same for argent xplorer domain)

Capture d’écran 2023-10-08 à 10 57 38

We also have to change the skeleton that does not fit anymore (`identitiesSkeleton.tsx`)

File: pages/identities.tsx [hunk 1]
```diff
@@ -8,12 +8,13 @@ import {
...
+import MintIcon from "../components/UI/iconsComponents/icons/mintIcon";
 import { useRouter } from "next/router";
 import { hexToDecimal } from "../utils/feltService";
 import IdentitiesSkeleton from "../components/identities/identitiesSkeleton";
 import TxConfirmationModal from "../components/UI/txConfirmationModal";
 import Wallets from "../components/UI/wallets";
+import ClickableAction from "../components/UI/iconsComponents/clickableAction";
 
 const Identities: NextPage = () => {
   const { account, address } = useAccount();
```
Comment (fricoben) [inline pages/identities.tsx]: You can add the fix made on the identity page here pls ? https://github.com/starknet-id/app.starknet.id/pull/424/files#diff-2f396cbcb2a07a00a226255f1fc151a03e18ef663d722f673002730415d82f4dR20

You'll be able to close the issue #423

**Review by fricoben — CHANGES_REQUESTED** [review]
Small change on the code side :)

**Review by Kevils — CHANGES_REQUESTED** [review]
Capture d’écran 2023-10-08 à 12 31 21
- Vertical gap between Title and description : 16px
- Vertical gap between title/description and CTA : 48px
- Description width max 650px


Capture d’écran 2023-10-08 à 12 37 47
- Vertical and horizontal gap between card identities : 32px
- Vertical gap between identities gap and CTA : 48px

image
- Au hover, ajouter 2px de stroke Primary et passer du Small shadow au Large shadow


Capture d’écran 2023-10-08 à 12 43 31
@fricoben Est-ce qu'on mettrait pas le numéro SID afin de card la même hauteur sur toutes les cards ?

Capture d’écran 2023-10-08 à 12 45 14
- Même retour que Benjamin sur les externals domains comme braavos et argent

**Review by Kevils — APPROVED** [review]

**Review by Kevils — CHANGES_REQUESTED** [review]
Capture d’écran 2023-10-08 à 15 01 12


- Fixé le width du CTA au responsive

File: components/identities/identitiesGalleryV1.tsx [hunk 2]
```diff
@@ -45,31 +45,39 @@ const IdentitiesGalleryV1: FunctionComponent<IdentitiesGalleryV1Props> = ({
...
+                : `ID: ${identity.id}`}
+            </p>
           </div>
         );
       })}
       {externalDomains.map((domain, index) => {
         const domainKind = getDomainKind(domain);
 
         function getIdentityImage(domainKind: DomainKind): ReactNode {
+          console.log(domainKind);
```
Comment (fricoben) [inline components/identities/identitiesGalleryV1.tsx]: Delete the console.log :)

**Review by fricoben — CHANGES_REQUESTED** [review]
On the front-end I have infinite loading when I'm not connected :) (easy to fix). 

Could we place this higher ? It's too much in the middle
Capture d’écran 2023-10-09 à 11 15 55

Could be cool to replace this by the image with the background (but without the border hover)

Capture d’écran 2023-10-09 à 11 16 22

We're almost done

**Comment by Marchand-Nicolas** [comment]
> On the front-end I have infinite loading when I'm not connected :) (easy to fix).

The issue come from the fix from here : https://github.com/starknet-id/app.starknet.id/pull/424/files#diff-2f396cbcb2a07a00a226255f1fc151a03e18ef663d722f673002730415d82f4dR20 I added. Is there something I missunderstood ? Because if I’m not wrong, in the fix it is explicitly written to load until the user connect its wallet

**Review by Kevils — APPROVED** [review]

**Comment by fricoben** [comment]
Capture d’écran 2023-10-10 à 10 26 09

If I'm not connected I still have an infinite loading

**Comment by fricoben** [comment]
After merge this PR should also close #423

**Review by fricoben — APPROVED** [review]
I fixed the connection problem, looks good to me

**Pull request merged**